src/components/NotesPage/NotesPageComponent.js

```diff
@@ -16,6 +18,8 @@ export class NotesPageComponent extends Component {
         <NotesContentEditorContainer />
         {!isHelpViewed && <ControlsHelpComponent onClose={closeHelp} />}
         <SearchPanelContainer />
+        <StatusButtonComponent />
```

Comment:
Status of what? Might make sense to name it `UploadsStatusButtonComponent`?

Reply:
will rename it